File: Tests/TestData/MultiFileCharacterization/SourceFiles/VbLibrary/Module1.vb
Imports System
Imports VbNetStandardLib

Module Module1
    Private dict As New Dictionary(Of Integer, Integer)

    Private Sub UseOutParameterInModule()
        Dim x
        dict.TryGetValue(1, x)
    End Sub

    Sub Main()
        Console.Write(AClass.NestedEnum.First)

        Dim interfaceInstance As AnInterface = New AnInterfaceImplementation
        Dim classInstance As New AnInterfaceImplementation
        Console.WriteLine(interfaceInstance.AnInterfaceProperty)
        Console.WriteLine(classInstance.APropertyWithDifferentName)

        interfaceInstance.AnInterfaceMethod()
        classInstance.AMethodWithDifferentName()
    End Sub

End Module

File: Tests/TestData/MultiFileCharacterization/VBToCSResults/ConvertWholeSolution/VbLibrary/Module1.cs
using System;
using System.Collections.Generic;
using Microsoft.VisualBasic.CompilerServices;
using VbNetStandardLib;

namespace VbLibrary
{
    static class Module1
    {
        private static Dictionary<int, int> dict = new Dictionary<int, int>();

        private static void UseOutParameterInModule()
        {
            var x = default(object);
            int argvalue = Conversions.ToInteger(x);
            dict.TryGetValue(1, out argvalue);
        }

        public static void Main()
        {
            Console.Write((int)AClass.NestedEnum.First);
            AnInterface interfaceInstance = new AnInterfaceImplementation();
            var classInstance = new AnInterfaceImplementation();
            Console.WriteLine(interfaceInstance.AnInterfaceProperty);
            Console.WriteLine(classInstance.AnInterfaceProperty);
        }
    }
}